docs/smt-input-language.rst

```diff
@@ -5,8 +5,8 @@ SMT-LIBv2 Input Language Reference
 .. highlight:: lisp
 
 This page contains a short description for the SMTLibv2 input language
-that STP can parse. For a longer description please read `this
-PDF <http://www.grammatech.com/resource/smt/SMTLIBTutorial.pdf>`__.
+that STP can parse. For more information related to SMTLIB, please refer to `this
```

Comment:
It should (probably) be "SMT-LIB" (with the hyphen). If you're touching this paragraph, could you also change "SMTLibv2" to "SMT-LIB v2" please?

Reply:
Done 🙂

Reply:
Thanks!